js/src/components/shipping-rate-section/minimum-order-card/edit-minimum-order-modal.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Form } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import AppModal from '.~/components/app-modal';
+import AppInputPriceControl from '.~/components/app-input-price-control';
+import VerticalGapLayout from '.~/components/vertical-gap-layout';
+import AppCountrySelect from '.~/components/app-country-select';
+
+const EditMinimumOrderModal = ( props ) => {
+	const { countryOptions, value, onChange, onRequestClose } = props;
+
+	const handleDeleteClick = () => {
+		onRequestClose();
+		onChange( {
+			countries: [],
+			currency: undefined,
```

Comment:
I just want to make the object have the same consistent schema as the form value object `{ countries, currency, threshold }`.

Reply:
Got it.

But if consider removing `currency` from `initialValues` due to https://github.com/woocommerce/google-listings-and-ads/pull/1266#discussion_r817609571, which is a similar concern mentioned in https://github.com/woocommerce/google-listings-and-ads/pull/1266#discussion_r819353999, then here could be removed, too.

And if aim to keep it, I just feel resetting `currency` to `undefined` looks buggy since currency should never be `undefined` during shipping setup.

Reply:
> I just want to make the object have the same consistent schema as the form value object `{ countries, currency, threshold }`.

Sorry, there was a slight mistake I said above. I actually meant to say I want to make the object have the same consistent schema as the `value` props. The idea is that the consumers pass a value into the component via `value` prop, and they get the new value from the component via `onChange` prop. I'd like the old value and the new value to have the same consistent schema.

> I just feel resetting `currency` to `undefined` looks buggy since currency should never be `undefined` during shipping setup.

I was thinking since this is a delete operation, it would make sense to have `currency: undefined` like `threshold: undefined`, i.e. everything is deleted.